dir.props

```diff
@@ -145,7 +145,7 @@
     <IsPrerelease>true</IsPrerelease>
 
     <!-- This should be kept in sync with package details in src/.nuget/init/project.json -->
-    <RuntimeIdGraphDefinitionVersion>1.0.2-beta-24224-02</RuntimeIdGraphDefinitionVersion>
+    <RuntimeIdGraphDefinitionVersion>1.1.0</RuntimeIdGraphDefinitionVersion>
```

Comment:
The `./build-packages.sh` command used in buildpipelines was failing looking for `./packages/*/1.0.2-beta-24224-02/runtime.json`.  My build was producing
```
ls -1 ./packages/*/*/runtime.json
./packages/microsoft.netcore.ilasm/2.1.0-preview3-26330-06/runtime.json
./packages/microsoft.netcore.jit/2.1.0-preview3-26330-06/runtime.json
./packages/microsoft.netcore.platforms/1.1.0/runtime.json
./packages/microsoft.netcore.runtime.coreclr/2.1.0-preview3-26330-06/runtime.json
./packages/microsoft.netcore.targets/1.1.0/runtime.json
```

My guess is this change is needed even w/o this PR.

Reply:
The linux-arm64 fails are due to this change.
Investigating further, I missed a step in buildpipelines in testing.  If I call the `./sync.sh` step this issue is resolved and I do not need this questionable change.  I'll force push a change to revert this line